.eslintrc.js

```diff
@@ -1,6 +1,9 @@
 module.exports = {
     'extends': 'airbnb',
+    'rules': {
+      'react/jsx-filename-extension': [1, { "extensions": [".js", ".jsx"] }],
```

Comment:
@jeffrey-xiao @adityavohra7 What would the recommendation be for a test that has jsx?  `.jsx` or `.js`? 

 I believe @jeffrey-xiao is suggesting tests can have jsx in `.js` files for tests, hence the whitelisted extension.  This generalized rule does seem to be ill suited for components and containers where we want to enforce jsx to be only in `.jsx` files.